.github/workflows/ci.yml

```diff
@@ -25,8 +29,10 @@ jobs:
           components: rustfmt, clippy
           override: true
 
-      - name: Install Dependencies
-        run: sudo apt-get install libssl-dev
+      - name: Install Dependencies (for ubuntu)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+            sudo apt-get install libssl-dev
```

Comment:
We also need macos & winodows to install dependency

Reply:
Apparently, this dependency is not required. I ran the workflow without installing libssl-dev on ubuntu-latest, and it ran fine - https://github.com/smrpn/actix-casbin-auth/pull/1/checks?check_run_id=2426521309. 
Is it necessary to include that as a dependency?
Apart from that, libssl-dev in openssl in macOS which is preinstalled.

Reply:
how about the windows platform

Reply:
![Screenshot from 2021-04-25 13-47-27](https://user-images.githubusercontent.com/55808717/115986228-dcb9af00-a5cc-11eb-9d93-27180cc532e8.png)
The workflow ran on windows without installing the dependency.

Reply:
lgtm